dev/tests/integration/testsuite/Magento/Catalog/Model/Product/UrlTest.php

```diff
@@ -42,6 +42,51 @@ public function testGetUrlInStore()
         $this->assertStringEndsWith('simple-product.html', $this->_model->getUrlInStore($product));
     }
 
+    /**
+     * @magentoDataFixture Magento/Store/_files/second_store.php
+     * @magentoConfigFixture default_store web/unsecure/base_url http://sample.com/
+     * @magentoConfigFixture default_store web/unsecure/base_link_url http://sample.com/
+     * @magentoConfigFixture fixturestore_store web/unsecure/base_url http://sample-second.com/
+     * @magentoConfigFixture fixturestore_store web/unsecure/base_link_url http://sample-second.com/
+     * @magentoDataFixture Magento/Catalog/_files/product_simple_multistore.php
+     * @magentoAppArea adminhtml
+     */
+    public function testGetUrlInStoreWithSecondStore()
```

Comment:
```suggestion
    public function testGetUrlInStoreWithSecondStore($storeCode, $expectedPorductUrl)
```
Please correct the test to match the correct dataprovider usage